docs/specs/git.yml

```diff
@@ -223,7 +223,7 @@ outputs:
 ---
 # Author can NOT be excluded if explicitly specified in YAML header, it's case insensitive
 repos:
-  https://github.com/testowner/testrepo#master-exclude-author-from-yaml-heaer:
+  https://github.com/testowner/testrepo#master-exclude-author-from-yaml-header:
```

Comment:
for the expected alert, we need to add those word to the allow list here :https://github.com/dotnet/docfx/blob/v3/.github/actions/spelling/expect.txt